CMakeLists.txt

```diff
@@ -332,7 +357,12 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
     DESTINATION ${CMAKE_INSTALL_PREFIX}/backends/pytorch
   )
 
-  FOREACH(plib ${PT_LIBS} ${OPENCV_LIBS})
+  install(
+    FILES       ${CMAKE_CURRENT_BINARY_DIR}/torchtrtc
```

Comment:
Run formatter. Extra space is not needed

Reply:
How do I do that ?

Reply:
https://github.com/triton-inference-server/server/blob/main/CONTRIBUTING.md#coding-convention elaborates how to install the necessary deps and run the formatter.

Reply:
Fixed manually.